docs/generated/packages/gradle/executors/gradle.json

```diff
@@ -24,6 +24,11 @@
         ],
         "description": "The arguments to pass to the Gradle task.",
         "examples": [["--warning-mode", "all"], "--stracktrace"]
+      },
+      "excludeDependsOn": {
```

Comment:
maybe it's worth adding that nx will still execute the dependencies here? Just so people aren't confused.

Why are we making this configurable anyways? Aren't we basically saying that nx might not catch all dependencies so we're giving this escape hatch to let gradle handle those cases? Why isn't nx always getting all dependencies correctly?

Reply:
changed